godot-macros/src/util/mod.rs

```diff
@@ -251,26 +251,28 @@ pub(crate) fn path_ends_with_complex(path: &venial::TypeExpr, expected: &str) ->
     })
 }
 
+pub fn is_cfg_attr(attr: &venial::Attribute) -> bool {
```

Comment:
This is a misleading name, I'd expect it checks for `cfg_attr` specifically, but it includes `cfg`.

```suggestion
pub fn is_cfg_or_cfg_attr(attr: &venial::Attribute) -> bool {
```